docs/AddingNewProjects.md

```diff
@@ -0,0 +1,44 @@
+# Adding New Projects To The Repo
```

Comment:
Can you add an entry for this new document to the table of contents in `docs/README.dm`?